src/vivarium_csu_alzheimers/components/__init__.py

```diff
@@ -2,7 +2,8 @@
 from .observers import (
     BaselineTestingObserver,
     BBBMTestCountObserver,
-    BBBMTestEligibilityObserver,
```

Comment:
The old one went from BBBMTestEligibilityObserver -> BBBMTestNewlyEligibleObserver.

The new one is then BBBMTestEligibleObserver

Reply:
Nit: you could have put all of these observations inside the same observer.

Reply:
Oh, duh